e2e/tests/notebook/reader-copy-to-workspace.spec.ts

```diff
@@ -29,24 +28,24 @@ describe('Workspace reader Jupyter notebook action tests', () => {
    * - Verify Workspace and notebook permissions.
    * - Copy notebook to collaborator Workspace.
    * - Full Edit rights to clone notebook.
-   * - Verify original notebook contents are unchaged in clone notebook.
```

Comment:
Note: this can easily be verified from the preview page, which does not require starting the runtime server.